builtin/common/after.lua

```diff
@@ -1,13 +1,16 @@
 local jobs = {}
 local time = 0.0
+local time_next = math.huge
 
 core.register_globalstep(function(dtime)
 	time = time + dtime
 
-	if #jobs < 1 then
+	if time < time_next then
 		return
 	end
 
+	time_next = math.huge
+
 	-- Iterate backwards so that we miss any new timers added by
 	-- a timer callback, and so that we don't skip the next timer
 	-- in the list if we remove one.
```

Comment:
This comment is now obsolete. The second reason is no longer relevant. The first reason is still valid, though; it implies that if a callback uses `minetest.after(0, ...)`, the new timer won't execute until the next globalstep.

Reply:
Done.